### PR TITLE
fix(fonts): update base font to 14px

### PR DESCRIPTION
**Pull request by mindreeper2420**
DO NOT MERGE

this PR updates the base font to 14 pixels.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 70c7e5102119de5855839555df18ae2afdcdccb1
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: src/assets/stylesheets/shared/_custom-variables.scss [hunk 1]
```diff
@@ -14,3 +14,4 @@ $color-pf-dark: #363636;
 $pf-icon-2x: 2em;
 $pf-icon-3x: 3em;
 $pf-icon-4x: 4em;
+$font-size-base: 14px;
```
Comment (pmuir) [inline src/assets/stylesheets/shared/_custom-variables.scss]: New line at end of file please

File: src/assets/stylesheets/shared/_overrides-for-patternfly.scss [hunk 2]
```diff
@@ -3,3 +3,9 @@
 {
   box-shadow: none !important;
 }
+.btn {
+  font-size: $font-size-base;
+}
+.form-control {
+  font-size: $font-size-base;
+}
```
Comment (pmuir) [inline src/assets/stylesheets/shared/_overrides-for-patternfly.scss]: new line at end of file please.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 70c7e5102119de5855839555df18ae2afdcdccb1
  * Your commit message does not contain a body

 * Commit: e620f799c4ea47e507e61d7f48ed7a1140b65ceb
  * Your commit message does not contain a body
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: src/assets/stylesheets/shared/_custom-variables.scss (hunk 1 above)
Comment (mindreeper2420) [inline src/assets/stylesheets/shared/_custom-variables.scss]: new line added

File: src/assets/stylesheets/shared/_overrides-for-patternfly.scss (hunk 2 above)
Comment (mindreeper2420) [inline src/assets/stylesheets/shared/_overrides-for-patternfly.scss]: new line added

**Comment by mindreeper2420** [comment]
@pmuir yes, these are in sync with the shared directory on Planner

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 70c7e5102119de5855839555df18ae2afdcdccb1
  * Your commit message does not contain a body

 * Commit: e620f799c4ea47e507e61d7f48ed7a1140b65ceb
  * Your commit message does not contain a body
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 6349e38d46bb5db50ffa0fad6bb9eabbded8e7a5
  * Your commit message does not contain a body
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by mindreeper2420** [comment]
[test]

**Comment by joshuawilson** [comment]
[test]

File: src/assets/stylesheets/shared/_custom-variables.scss (hunk 1 above)
Comment (joshuawilson) [inline src/assets/stylesheets/shared/_custom-variables.scss]: @mindreeper2420 it doesn't look like there is new line at the end.

Comment (mindreeper2420) [inline src/assets/stylesheets/shared/_custom-variables.scss]: @joshuawilson that's odd, as it looks like there is a new line for me when I look at all of the commits combined

**Comment by joshuawilson** [comment]
@mindreeper2420 what version of Planner did your changes get merged into?

**Comment by joshuawilson** [comment]
@mindreeper2420 please change the commit text to match the required Semantic Release syntax. You can find a reference in the README.

**Comment by mindreeper2420** [comment]
@joshuawilson it has not been merged into Planner yet - still waiting on reviews to complete. It will need to be rebase into Planner Master, as 5 other PRs were merged in since I submitted mine.

**Comment by mindreeper2420** [comment]
@joshuawilson what needs to change for the commit text? Everything seems to say it needs a body, when it has one :-/

**Comment by joshuawilson** [comment]
@mindreeper2420 you have 3 commits, one has "fix(fonts): ..." the others don't. The body is done by skipping a line and adding text starting on the 3rd line. There is also a footer if you want that is done be skipping a line after the body.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 70c7e5102119de5855839555df18ae2afdcdccb1
  * Your commit message does not contain a body

 * Commit: 9d1737da19627364ed446ffda1fd361e58604296
  * Your commit message does not contain a body

 * Commit: b0f8b7bcea40c822132c71d797be88fe5264972c
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by mindreeper2420** [comment]
@joshuawilson fixed

**Comment by joshuawilson** [comment]
[test]

**Comment by pmuir** [comment]
[test]

**Comment by joshuawilson** [comment]
@mindreeper2420 can we merge this without the changes in planner being merged?

**Comment by mindreeper2420** [comment]
@joshuawilson - yes, should be fine. Planner PR was approved and rebate, so that should be in soon.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 70c7e5102119de5855839555df18ae2afdcdccb1
  * Your commit message does not contain a body

 * Commit: 9d1737da19627364ed446ffda1fd361e58604296
  * Your commit message does not contain a body

 * Commit: b0f8b7bcea40c822132c71d797be88fe5264972c
  * Your commit message does not contain a body

 * Commit: cc779661a9ba39419b96c92b3e3143bbda749612
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7e32f4a6ee546843f1477dd76031c9e0a3375a01
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7e32f4a6ee546843f1477dd76031c9e0a3375a01
  * Your commit message does not contain a body

 * Commit: 92477c41f2c79b9a6cbfd264491ae5035d29ee94
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 048385321b12ac8bfa91cd1a7e5e5cde474fc9fb
  * Your commit message does not contain a body

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**